src/Microsoft.DotNet.XHarness.CLI/Commands/WASM/WasmTestCommand.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using Microsoft.DotNet.XHarness.CLI.CommandArguments;
+using Microsoft.DotNet.XHarness.CLI.CommandArguments.Wasm;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.DotNet.XHarness.CLI.Commands.Wasm
+{
+    internal class WasmTestCommand : TestCommand
+    {
+        private readonly WasmTestCommandArguments _arguments = new WasmTestCommandArguments();
+
+        protected override TestCommandArguments TestArguments => _arguments;
+
+        protected override string CommandUsage { get; } = "wasm test [OPTIONS]";
+
+        protected override string CommandDescription { get; } = "Executes BCL xunit tests on WASM";
+
+        protected override Task<ExitCode> InvokeInternal(ILogger logger)
+        {
+            logger.LogDebug($"Wasm Test command called");
```

Comment:
@MaximLipnin this gets called when you call the `wasm test` command. Because you didn't supply `-v`, it didn't show

Reply:
The same with -v:
```
dotnet Microsoft.DotNet.XHarness.CLI.dll wasm test -v -o=out         
XHarness command issued: wasm test -v -o=out
XHarness exit code: 1001
```

Reply:
@MaximLipnin please merge newest master, there is a fix connected to that. However that is the place where you should put your code to start things